api/types/network/network.go

```diff
@@ -13,6 +13,13 @@ type IPAM struct {
 	Config  []IPAMConfig
 }
 
+// Options represents the network options for endpoint creation
+type Options struct {
```

Comment:
not really i guess. Since this is part of the network type package it would be evident. Not particular about it. I can change if required.